build-check.txt

```diff
@@ -0,0 +1,2 @@
+Integrity checks failed:
+#<struct Commons::Integrity::Check::Base::Error category=:position_id_message, message="Q132050 was found in boundaries/build/index.json but not in boundaries/build/position-data.json", filename=#<Pathname:boundaries/build/position-data.json>>
```

Comment:
Is this just an area we lack data for?

Reply:
The boundaries index should use a specific position item ID, not just 'governor', so yeah, it needs fixing :-)

Reply:
Oh, wait. The boundary entry is for the FLACS of the FATA, which has gone, so can just be removed.